Automattic/wp-calypso
# Logging: Render site log data in a table

Fixes Automattic/dotcom-forge#1989

## Proposed Changes

* Renders log entries in a table
* Scrolls horizontally
* Changes the Sites Log section to a "full width" layout
* `date`/`timestamp` column (depending on the log type) appears as first column
* The first column always appears in the site's configured timezone


![CleanShot 2023-03-24 at 18 11 05@2x](https://user-images.githubusercontent.com/1500769/227430276-36fc7ac2-9555-408f-b4b4-2ff73884eddb.png)

I decided to use the site's timezone (rather than the current user's timezone) because I thought it might be nicer to have consistency between team members. For example if an HE is helping a user track down a bug, it might be better for everyone to be using the same frame of reference when talking.
Whichever timezone is used personally I prefer when the timezone is explicitly displayed too. To avoid confusion.

## Testing Instructions



* Check out locally
* Head to `/site-logs/:siteSlug` for a WoA site
* Take a look at the table
* Acceptance criteria in Automattic/dotcom-forge#1989 should be complete
* You can change the site's timezone at `/settings/general/:siteSlug`

## Pre-merge Checklist
